### ISSUE TITLE
Mitigate accessibility issues

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#156]
1. Placing menu items when clicked on more options
2. Empty screen images - These should be embedded and should not be recognized by the voiceover except for the text and the button itself.
4. Make sure text is aligned in center on Level-2, Settings
5. Double-check accessibility on Settings screen to make sure it is reading headings properly.
6. Double-check accessibility layout on text oriented screens, make sure they are using proper labels to avoid confusion and grammatical errors

**Comment by Rajat-Sen** [comment]
Adding the images for reference!
<img width="393" alt="Settings-screen" src="https://user-images.githubusercontent.com/38566088/149025281-b82ef3d8-d0ab-4b92-896c-b0ac3195e0be.png">

**Comment by Rajat-Sen** [comment]
<img width="194" alt="Help  Faqs" src="https://user-images.githubusercontent.com/38566088/149029570-f1597175-0d76-4768-9f49-b02b18047f26.png">

**Comment by ForumG-SC** [comment]
@Rajat-Sen Revalidate settings screen after new deployment is done.

**Comment by Rajat-Sen** [comment]
Validated!
